database/database.go

```diff
@@ -26,6 +27,11 @@ type IDatabaseService interface {
 	GetLatestValidatorRegistrations(timestampOnly bool) ([]*ValidatorRegistrationEntry, error)
 	GetValidatorRegistration(pubkey string) (*ValidatorRegistrationEntry, error)
 	GetValidatorRegistrationsForPubkeys(pubkeys []string) ([]*ValidatorRegistrationEntry, error)
+	// IsMevCommitValidatorRegistered(pubkey string) (bool, error)
```

Comment:
Note: I've decided to avoid using the DB entirely, as we only need to store data temporarily. So all data will be stored in redis.

Reply:
I'll be removing all the DB changes.

Reply:
Keeping the commented out aspects, in case @aloknerurkar @mrekucci @shaspitz think it's a good idea to persist this data. The reason for persisting is to ensure we have safety, in case redis invalidates the data too quickly. However, given both the builder and validator registrations are getting updated in redis every slot and 32 slots respectively, it doesn't make sense to persist the data, especially given it's ephemeral nature.